### PR TITLE
WIP: Update draft-irtf-t2trg-rest-iot.md

### PR DESCRIPTION
**Pull request by mkovatsc**
Currently work in progress

### CODE REVIEW

File: draft-irtf-t2trg-rest-iot.md [hunk 1]
```diff
@@ -283,16 +281,18 @@ Uniform Resource Identifier (URI):
...
-Note that the terms "client" and "server" refer only to the roles that the nodes assume for a particular message exchange. The same node might act as a client in some communications and a server in others.
+Note that the terms "client" and "server" refer only to the roles that the nodes assume for a particular message exchange.
+The same node might act as a client in some communications and a server in others.
 Classic user agents (e.g., Web browsers) are always in the client role and have the initiative to issue requests.
 Origin servers always have the server role and govern over the resources they host.
 Simple IoT devices, such as sensors and actuators, are commonly acting as servers and exposing their physical world interaction capabilities (e.g., temperature measurement or door lock control capability) as resources.
-Typical IoT system client can be a cloud service that retrieves data from the sensors and commands the actuators based on the sensor information.
+Typical IoT system clients can be a cloud service that retrieves data from the sensors and commands the actuators based on the sensor information.
```
Comment (akeranen) [inline draft-irtf-t2trg-rest-iot.md:295]: Should also "cloud services" be plural here?

**Review by akeranen — APPROVED** [review]

File: draft-irtf-t2trg-rest-iot.md [hunk 2]
```diff
@@ -427,9 +424,9 @@ Typical media-types for IoT systems include:
...
 * "application/senml+json" and "application/senml+cbor" for Sensor Measurement Lists (SenML) data {{RFC8428}}
 
-A full list of registered Internet Media Types is available at the IANA registry {{IANA-media-types}} and numerical identifiers for media-types, parameters, and content-codings registered for use with CoAP are listed at CoAP Content-Formats IANA registry {{IANA-CoAP-media}}.
+A full list of registered Internet Media Types is available at the IANA registry {{IANA-media-types}} and numerical identifiers for media types, parameters, and content codings registered for use with CoAP are listed at CoAP Content-Formats IANA registry {{IANA-CoAP-media}}.
 
-The terms "media-type", "content-type", and "content-format" (short identifier of content-type and content-coding, abbreviated for historical reasons "ct") are often used when referring to representation formats used with CoAP.
+The terms "media type", "content type" (media type plus potential parameters), and "content format" (short identifier of content type and content coding, abbreviated for historical reasons "ct") are often used when referring to representation formats used with CoAP.
```
Comment (mkovatsc) [inline draft-irtf-t2trg-rest-iot.md:429]: This statement is pretty vague considering that it took us a long time to cleanly separate these terms (cf. "ct"). I added the term "Content Type" to terminology.

File: draft-irtf-t2trg-rest-iot.md [hunk 3]
```diff
@@ -387,37 +384,37 @@ Uniform Resource Identifiers (URIs) are used to indicate a resource for interact
...
 Constrained Resource Identifiers {{!I-D.ietf-core-href}} provide a CBOR-based format of URIs that is better suited also for resource constrained IoT devices.
 
 ## Representations
 
-Clients can retrieve the resource state from an origin server or manipulate resource state on the origin server by transferring resource representations.
-Resource representations have a content-type (media-type, optionally with parameters) that tells how the representation should be interpreted by identifying the representation format used.
-
-Typical media-types for IoT systems include:
+Clients can retrieve the resource state from a server or manipulate resource state on the (origin) server by transferring resource representations.
+Resource representations must have metadata that identies the representation format used, so the representations can be interpreted correctly.
```
Comment (mkovatsc) [inline draft-irtf-t2trg-rest-iot.md:415]: Wiggling around the media type vs content type vs content format topic here...

**Comment by mkovatsc** [comment]
> 
> 
> Hmm. Wasn't that useful information?

Carsten's comment refers to the strong reduction of the "Thing" definition.
I tried to get some useful information back, but kept the started discussion on "Thingness" out.

**Comment by mkovatsc** [comment]
The term "affordance" was actually never used in the text (only in terminology). I added more information that puts hypermedia controls in context to affordances.

**Comment by mkovatsc** [comment]
We might have a bit of a problem as the work on CoRAL stalled with Klaus not being active on it anymore...

@ektrah Might this change now after the change on your side I heard about?

File: draft-irtf-t2trg-rest-iot.md (hunk 1 above)
Comment (mkovatsc) [reply draft-irtf-t2trg-rest-iot.md]: I took the too easy fix... needed a change to "A typical ..." -- done now.

**Pull request merged**